docs/ErrorHandling.rst

```diff
@@ -52,10 +52,10 @@ for developers to want to handle errors from different operations in
 the same basic way, either by reporting the error to the user or
 passing the error back to their own clients.
 
-These errors will be the focus on this proposal.
+These errors will be the focus of this proposal.
 
 The final two classes of error are outside the scope of this proposal.
-A **universal error** is theoretically recoverable, but by its nature
+A **universal error** is theoretically recoverable, but by its nature,
```

Comment:
Extraneous comma.  If you wish to make the `but` parenthetical then move it there and reflow this sentence.